extensions/wikia/SpecialNewWikis/SpecialNewWikis_body.php

```diff
@@ -169,9 +169,11 @@ function getQueryInfo() {
 	}
 	/* need to be overwrite because of not unique col name (city_id) */
 
-	function reallyDoQuery( $offset, $limit, $descending ) {
+	function reallyDoQuery( $offset, $limit, $descending, $info = [] ) {
```

Comment:
This will cause `PHP Warning` because the method declaration will not be compatible with the super method declaration.

Reply:
We've added $info = [] to other methods, so it won't warn.

Reply:
It will - parent method in `IndexPager` superclass does not take `$info` parameter.